depends/packages/native_sip.mk

```diff
@@ -0,0 +1,18 @@
+package=native_sip
+$(package)_version=4.16.6
```

Comment:
Just updated to 4.16.7. May want to consider updating. Same for other packages, although I suppose it's not a major issue so long as later versions don't fix major bugs or security issues.